internals/secrethub/service_aws_init.go

```diff
@@ -56,6 +56,10 @@ func (cmd *ServiceAWSInitCommand) Run() error {
 		return err
 	}
 
+	if cmd.role == "" && cmd.kmsKeyID == "" {
+		fmt.Fprintln(cmd.io.Stdout(), "This command creates a new service account for use on AWS. For help on this, see https://secrethub.io/docs/reference/service-command/#command-service-aws-init.")
```

Comment:
This is a dead link at the moment. Also, shouldn't you say that certain flags are missing?

Reply:
> Also, shouldn't you say that certain flags are missing?

You will be prompted for these values later. As users should be familiar with the concept of AWS services in order to know what values to supply, we thought the help should be referenced for users that have not created an AWS service before.
When the flags are supplied, you probably know about the concept, as you know what values you want to use.

Reply:
> This is a dead link at the moment.

Should we reference a link or just add a longer help-text to `secrethub service aws init` (with long help)?

Reply:
A suggestion for a long help text:
```
The native AWS identity provider uses a combination of AWS IAM and AWS KMS to provide access to SecretHub for any service running on AWS (e.g. EC2, Lambda or ECS). For this to work, an IAM role and KMS key are needed. The role should be the role that is assumed by the service during executing: the "IAM Role" for an EC2 instance, "Task Execution Role" for an Lambda Function and "Task Role" for ECS Tasks.

The KMS key is a key that is used for encrypting the account key of the service. Decryption permission on this key must be granted to the previously described IAM Role.

To create a new service that uses the AWS identity provider, the CLI must have encryption access to the KMS key that will be used by the service account. Therefore AWS credentials should be configured on this system. For details on how this can be done, see https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html.

If no system-wide default for the AWS region is provided (e.g. with $AWS_REGION), the AWS-region where the KMS key resides should be explicitly provided to this command.
```

Reply:
Amazing! Very concise and clear. I made a few grammatical and style fixes to your text. (shown in bold)

> The native AWS identity provider uses a combination of AWS IAM and AWS KMS to provide access to SecretHub for any service running on AWS (e.g. EC2, Lambda or ECS). For this to work, an IAM role and **a KMS** key are needed. 
>
> - The role should be the role that is assumed by the service during **execution**: the "IAM Role" for an EC2 instance, "Task Execution Role" for an Lambda Function and "Task Role" for ECS Tasks.
> - The KMS key is a key that is used for encrypting the account key of the service. Decryption permission on this key must be granted to the previously described IAM Role.
>
> To create a new service that uses the AWS identity provider, the CLI must have encryption access to the KMS key that will be used by the service account. Therefore AWS credentials should be configured on this system. For details on how this can be done, see https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html.
>
> If no system-wide default for the AWS region is provided (e.g. with $AWS_REGION), the AWS-region where the KMS key resides should be explicitly provided to this command **with the --region flag.**

Small consideration, we can make the second bullet even shorter by saying: `The KMS key is a key that is used for encryption of the account.` Not necessary, but worth thinking over for a sec.

Reply:
Good changes. I think it's worth mentioning that the role should have access. This can prevent a nasty headache.